package.json

```diff
@@ -55,27 +55,26 @@
     "cssnano": "3.10.0",
     "fs-extra": "1.0.0",
     "git-rev-sync": "1.4.0",
-    "grunt": "0.4.5",
-    "grunt-cli": "0.1.13",
-    "grunt-contrib-connect": "0.11.2",
-    "grunt-contrib-copy": "0.8.0",
+    "grunt": "^1.0.3",
+    "grunt-cli": "^1.3.1",
+    "grunt-contrib-connect": "^2.0.0",
+    "grunt-contrib-copy": "^1.0.0",
     "grunt-contrib-jasmine": "1.1.0",
```

Comment:
I'm sure you discovered this, but upgrading to `grunt-contrib-jasmine` to `2.0.2` gives us a log of benefits:
- es6 support
- Chrome headless

Of course, this requires a bit of updating on our end (I took a quick look at it a few months back with #520) - but something we should explore soon!

Reply:
for sure! both (move to webpack and upgrade jasmine plus karma) are all in radar!
and I think upgrade grunt plugin would be helpful at paving the way.